integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/service/CompanyCertificateService.java

```diff
@@ -168,6 +168,26 @@ public String getCommonNameFromApplicationCertificate(Application application) {
   }
 
   /**
+   * Get a email address from the application certificate
+   * @param application Application object
+   * @return EmailAddress from the application certificate or empty string if the certificate doesn't exist
+   */
+  public String getEmailAddressFromApplicationCertificate(Application application) {
```

Comment:
Include one unit test to validate the flow when the certificate file has not been found.